### PR TITLE
replaced name@ with info@

### PR DESCRIPTION
**Pull request by jnakaso**
## What

> changed name@example.com to info@openseattle.org

## Why Do

> Use the correct emails

### CODE REVIEW

**Comment by santina** [comment]
@ybalenko looks like Jeff was already on it :)

File: pages/_app.tsx [hunk 1]
```diff
@@ -6,12 +6,9 @@ import { AppProps } from 'next/app'
 import { theme } from 'theme/theme'
 
 export default function App({ Component, pageProps }: AppProps) {
-  console.log(pageProps)
```
Comment (santina) [inline pages/_app.tsx:9]: thank you!

File: components/CommonFooter.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const CommonFooterLargeScreen = () => (
     <GridItem md={4}>
       <FooterItemTitle>Contact us</FooterItemTitle>
       <FooterSubItem>
-        <a href="mailto:name@email.com">info@openseattle.org</a>
+        <a href="mailto:info@openseattle.org">info@openseattle.org</a>
```
Comment (santina) [inline components/CommonFooter.tsx:67]: curious where does this email go to?

Comment (santina) [reply components/CommonFooter.tsx:67]: whom*

**Review by santina — APPROVED** [review]

File: components/CommonFooter.tsx (hunk 2 above)
Comment (jnakaso) [reply components/CommonFooter.tsx:67]: Seamus has many aliases :-)

**Comment by ybalenko** [comment]
Verified in https://open-seattle-website-git-jeffn-footer-em-9bc958-dev-openseattle.vercel.app/ . Works as expected! Thank you.

**Pull request merged**